portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,16 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact about me to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  const facts =
+      ['I like seafood', 'I like soccer', 'I like coding', 'I like joking'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick a random fact about me.
+  const fact = greetings[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const factContainer = document.getElementById('fact-container');
```

Comment:
If you're changing which element you're fetching from the DOM, I would expect to see a corresponding change in the HTML where the element "id" attribute is changed from "greeting-container" to "fact-container" there as well.

Reply:
Yes absolutely my bad. I think i accidentally made the pull request when it was not ready.